portal_objects/metaworkflows/SV_germline.json

```diff
@@ -217,6 +217,29 @@
         }
       ]
     },
+
+    {
+      "name": "workflow_manta_add_confidence",
+      "workflow": "0020ca8d-f826-480f-a78a-e34fb7961dde",
+      "config": {
+        "instance_type": "t3.small",
+        "ebs_size": "2x",
+        "EBS_optimized": true,
+        "spot_instance": true,
+        "run_name": "run_workflow_manta_add_confidence",
+        "behavior_on_capacity_limit": "wait_and_retry"
+      },
+      "input": [
+        {
+          "argument_name": "input_vcf",
+          "argument_type": "file",
+          "source": "workflow_annotateSV_sansa_vep_vcf-check",
+          "source_argument_name": "annotated_SV_vcf"
+        }
+      ]
```

Comment:
we need to add 

      "custom_pf_fields": {
          "vcf_sv_confidence": {
              "variant_type": "SV",
              "file_type": "confidence vcf",
              "description": "intermediate file"
          }